gen/build_deploy/bash.py

```diff
@@ -17,6 +18,21 @@
 from pkgpanda.util import logger
 
 
+def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
+    if calculate_fault_domain_enabled(fault_domain_detect_filename) == 'false':
```

Comment:
Instead of calling the calculate function again, you can reuse its result. Add `fault_domain_enabled` as an argument to this function and gen will make sure that it gets passed in.